server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -317,7 +388,7 @@ public void start() {
 
     if (Persistence.hasDB()) {
       initializeCounties();
-      initializeASM();
+      initializeASMs();
```

Comment:
Or, rather, without _known valid_ ones. If we get a `PersistenceException` there, it means either "something has gone wrong and we can't talk to the database", or "I found inconsistent information in the database that makes me unable to determine the states of the ASMs" (such as two rows describing the same ASM). I could probably eliminate that possibility by making the primary key of that table be based on the class and identity of the ASM, but that's just pushing the problem elsewhere.